kable-core/src/commonMain/kotlin/Peripheral.kt

```diff
@@ -153,6 +153,15 @@ public interface Peripheral : CoroutineScope {
      */
     public val services: StateFlow<List<DiscoveredService>?>
 
+    /**
+     * Return the current ATT MTU size, minus the size of the ATT headers (3 bytes).
+     *
+     * On Android, this will be the default (23 - 3) unless you called [requestMtu] when connecting.
+     * For iOS, this is automatically negotiated, and can also vary depending on the writeType.
+     * On JavaScript, this will return the default (23 - 3) every time as there is no ATT MTU property available.
+     */
+    public suspend fun maximumWriteValueLengthForType(writeType: WriteType): Int
```

Comment:
I'm torn on the best approach for getting MTU:
1. via `suspend` function (as you have here)
2. via a flow (as [currently implemented for Android](https://github.com/JuulLabs/kable/blob/61bfcb87b610a1e6f96fd03a1865f53bf6a2ec43/kable-core/src/androidMain/kotlin/AndroidPeripheral.kt#L157-L162))

The current Android implementation provides the raw MTU, but I'm not aware of any use-cases that need the raw value, so I think it makes more sense to only provide the value as you have it here (maximum write size).

The Android flow implementation also should be moved to common code (on the `Peripheral` as you have it here).

Do you have any experience in needing to know the max write size that favors one approach over the other?

Reply:
The reason I didn't use the Flow is that the MTU should never be changed. According to the BLE spec, you should negotiate (request) the MTU immediately when connected. This is also why iOS only have this method instead of a callback.

Reply:
Makes sense. 👍 

Outside the scope of this PR, but I'll plan on dropping `requestMtu` function from `AndroidPeripheral` and instead have MTU configured via the peripheral builder.